lib/UserAgentApplication.ts

```diff
@@ -183,24 +183,28 @@ namespace Msal {
         * @param _tokenReceivedCallback -  The function that will get the call back once this API is completed (either successfully or with a failure).
         * @param {boolean} validateAuthority -  boolean to turn authority validation on/off.
         */
-        constructor(clientId: string, authority: string, tokenReceivedCallback: tokenReceivedCallback, validateAuthority?: boolean) {
+        constructor(
+            clientId: string,
+            authority: string,
+            tokenReceivedCallback: tokenReceivedCallback,
+            validateAuthority?: boolean,
+            cacheLocation = 'sessionStorage'
+        ) {
+            this.cacheLocation = cacheLocation
             this.clientId = clientId;
-
             this.validateAuthority = validateAuthority === true;
-            this.authority = authority ? authority : "https://login.microsoftonline.com/common";
+            this.authority = authority || 'https://login.microsoftonline.com/common';
```

Comment:
The condition you removed does the same thing. Any particular reason for this change.